DBoW2/DBoW2_sample_code.cpp

```diff
@@ -0,0 +1,218 @@
+#include "DBoW2.h"
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/features2d.hpp>
+#include <easy/profiler.h>
+
+#include <iostream>
+#include <vector>
+#include <filesystem>
+#include <string>
+#include <memory>
+
+using namespace DBoW2;
+namespace fs = std::filesystem;
+namespace UnitTest
+{
+  using FeatureVector = std::vector<std::vector<cv::Mat>>;
+  using Features = std::vector<cv::Mat>;
+  using FileLUT = std::vector<std::string>;
+}
+
+bool loadDBFeatures(UnitTest::FeatureVector &vfeatures, UnitTest::FileLUT &dbTable, const cv::Ptr<cv::ORB> &orb);

```

Comment:
```suggestion
bool loadDBFeatures(UnitTest::FeatureVector& vfeatures, UnitTest::FileLUT& dbTable, const cv::Ptr<cv::ORB>& orb);

```

저희 코딩 가이드라인에서는 &는 변수명이 아닌 타입에 붙습니다. (이 쪽이 좀 더 가독성 높아요! :smile_cat: )

아래 다른 함수들에도 적용해주시면 좋을 것 같습니다